react/useAddressRules.tsx

```diff
@@ -0,0 +1,28 @@
+import { useMemo } from 'react'
+import { useQuery } from 'react-apollo'
+
+import addressRulesQuery from './graphql/addressRulesQuery.gql'
+
+const useAddressRules = () => {
+  const { data } = useQuery(addressRulesQuery)
+
+  const addressRules = useMemo(
+    () =>
+      data?.allCountriesData.reduce(
+        (rules: any, countryData: any) => ({
```

Comment:
I know this is just copy pasted, but can we replace the `any` to the correct type? I tried it but it isn't trivial ><